firebase/test/firestore-rules/setup.ts

```diff
@@ -0,0 +1,6 @@
+import * as utils from "./utils";
+
+// Clears all firestore data after each test
+afterEach(async () => {
```

Comment:
This works even not being referenced by the other files?

Reply:
This through mocha command line, called in the test script.